sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/util/ListModelOptions.java

```diff
@@ -35,8 +35,9 @@ public ListModelOptions setMaxItemCount(Integer maxItemCount) {
 
     /*
      * Whether to include the model definition in the result. If false, only the model metadata will be returned.
+     * Enabled by default.
      */
-    private Boolean includeModelDefinition;
+    private Boolean includeModelDefinition = true;
```

Comment:
Will this be included retrieving multiple models also?

Reply:
yes, it's only used for the list operation and it is turned on by default in .NET, so I turned it on by default in here too.

Reply:
Isn't this false by default, on the .NET client?

Reply:
no, On by default.

Reply:
Correction: 
On GetModel we use default value of true and do not have an overload to change it. So this constant will not change. 
On ListModel, default value is false.